runtime/compiler/optimizer/J9Optimizer.cpp

```diff
@@ -426,6 +428,7 @@ static const OptimizationStrategy reducedWarmStrategyOpts[] =
    { OMR::globalDeadStoreElimination,                OMR::IfVectorAPI           }, // global dead store removal
    { OMR::deadTreesElimination,                      OMR::IfVectorAPI           }, // cleanup after dead store removal
    { OMR::vectorAPIExpansion,                        OMR::IfVectorAPI},
+   { OMR::osrGuardRemoval,                       OMR::IfVectorAPI },
```

Comment:
Please add whitespaces before and after `OMR::IfVectorAPI` to align this line with other lines.